lib/msal-angular/src/msal.guard.ts

```diff
@@ -139,8 +143,9 @@ export class MsalGuard implements CanActivate, CanActivateChild, CanLoad {
         return this.activateHelper(state);
     }
 
-    canLoad(): Observable<boolean|UrlTree> {
+    canLoad(): Observable<boolean> {
         this.authService.getLogger().verbose("Guard - canLoad");
+        // @ts-ignore
```

Comment:
Instead of ignoring the compiler errors can we force the return of boolean on the return? 
`return !!this.activateHelper()`

Reply:
No, because we want the routetree returned for Angular 10 and up.

Reply:
We're just telling Angular that it will be a boolean, to allow it to have the function signature that is compatible with Angular 9.

Reply:
hmm ok, not the biggest fan of using ts-ignore but if it's unavoidable I suppose it's fine for now